tests/usecase/test_check_terminate_process_on_crashing_app.py

```diff
@@ -0,0 +1,15 @@
+import basetest
+
+class TestCaseTerminateChildProcessesCompleteOnCrashingApp(basetest.BaseTest):
+
+    def setUp(self):
+        self.setUpCF('Sample-StartError.mpk', env_vars={
+            'DEPLOY_PASSWORD': self.mx_password,
+            'METRICS_INTERVAL': '10',
+        })
+        self.startApp()
+
+    def test_verify_termination_logs(self):
```

Comment:
I don't think this tests the actual error, which is that the state is still in starting mode, even though startup was aborted and start.py did exit(1).

My suggestion would be to check for the log line `start failed, stopping`, sleep 10 seconds and then see if instance state is not equal to `STARTING` in `cf curl /v2/apps/a0f891be-109f-4313-b542-7e5df6acf110/instances`

Reply:
Moved invocation of `startApp` from set-up to the test case method itself. 
The test case now also checks for the string when aborting is happen to stop the buildpack process.